src/reducers/timerReducer.js

```diff
@@ -12,12 +12,7 @@ export default function(state = initialState, action) {
             return {
                 ...state,
                 timerShouldRun: action.payload
```

Comment:
Here is what I referenced earlier, instead of using `action.payload` you could use `!state.timerShouldRun`. It's not really that big of a deal, but I think this is a standard redux convention

Reply:
This makes sense. In learning about redux I have heard many times that "actions dispatch to reducers who in turn defines what actually changes". Or something like that. I have updated the logic for increment timer too to follow this convention. Am I on to something?